x/profile/keeper/msg_server_delete_coordinator.go

```diff
@@ -0,0 +1,33 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+func (k msgServer) DeleteCoordinator(
+	goCtx context.Context,
+	msg *types.MsgDeleteCoordinator,
+) (*types.MsgDeleteCoordinatorResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Check if the coordinator address is already in the store
+	coordByAddress, found := k.GetCoordinatorByAddress(ctx, msg.Address)
+	if !found {
+		return &types.MsgDeleteCoordinatorResponse{},
+			sdkerrors.Wrap(types.ErrCoordAddressNotFound, msg.Address)
+	}
+
+	coord := k.GetCoordinator(ctx, coordByAddress.CoordinatorId)
+	if (coord == types.Coordinator{}) {
```

Comment:
Same comment as `MsgUpdateCoordinatorDescription` we must use `found` boolean

Reply:
Sorry for the ambiguous response. I actually realize that unlike `map`, for `list` we don't have a `found` boolean returned. We must use the method `k.HasCoordinator` instead to check for coordinator existence (same for other PRs)